root/layout/header/bottom.tt

```diff
@@ -9,9 +9,12 @@
 
 <ul class="menu">
     <li class="about">
-        <a href="[% doc_link('About') %]">[% l('About') %]</a>
+        <span class="menu-header">[% l('About Us') %]</span>
         <ul>
             <li>
+                <a href="[% doc_link('About') %]">[% l('About MusicBrainz') %]</a>
+            </li>
```

Comment:
Unsure of the best name for this. "About MusicBrainz"? Just "MusicBrainz"? I didn't want to repeat "About Us".